src/client/background.rs

```diff
@@ -0,0 +1,56 @@
+use futures::{Future, Poll};
+use hyper::body::Payload;
+use hyper::client::conn::Connection as HyperConnection;
+use std::fmt::{self, Debug};
+use tokio_io::{AsyncRead, AsyncWrite};
+
+use log::error;
```

Comment:
can we group this with the others up top?

Reply:
OK. This one felt special because: 1) it's a macro import; 2) I'd like it to go away eventually (see #45), so I wanted it to stick out like a sore thumb.